src/main/webapp/app/overview/postings/postings.component.ts

```diff
@@ -140,8 +140,8 @@ export class PostingsComponent implements OnInit, AfterViewInit {
     }
 
     private updatePostAfterVoteChange(post: Post): void {
-        const indexToUpdate = this.posts.findIndex((post) => {
-            return post.id === post.id;
+        const indexToUpdate = this.posts.findIndex((p) => {
+            return p.id === p.id;
```

Comment:
We have a policy to not use one letter variables (or at least we try to pursue this approach), could you change the `findIndex` variable to something like `currentPost` or change the method parameter to `upvotedPost`

Reply:
ok I change it. Also when sorting? This is used directly above: 
```    
private static sortPostsByVote(posts: Post[]): Post[] {
        return posts.sort((a, b) => {
            return b.votes! - a.votes!;
        });
    }
```

Reply:
Good question. You could call it `postA` and `postB`, but we actually do often call the sort method with `a` and `b`, so maybe you could say this is sort of a convention.

Reply:
+ question: shouldn't it be `p.id === upvotedPost.id` ?

Reply:
thanks...